go.sum

```diff
@@ -0,0 +1,51 @@
+github.com/BurntSushi/toml v0.3.1/go.mod h1:xHWCNGjB5oqiDr8zfno3MHue2Ht5sIBksp03qcyfWMU=
+github.com/Clever/go-utils v0.0.0-20180917210021-2dac0ec6f2ac h1:eoofDGlVjiTro1kr97QnRkW4b/MGDSmm4E0ta2rqFos=
+github.com/Clever/go-utils v0.0.0-20180917210021-2dac0ec6f2ac/go.mod h1:XXTA9F/953CGhnDRslFaBp7S0560745gMZSDm+C4ebw=
+github.com/armon/consul-api v0.0.0-20180202201655-eb2c6b5be1b6/go.mod h1:grANhF5doyWs3UAsr3K4I6qtAmlQcZDesFNEHPZAzj8=
+github.com/coreos/etcd v3.3.10+incompatible/go.mod h1:uF7uidLiAD3TWHmW31ZFd/JWoc32PjwdhPthX9715RE=
+github.com/coreos/go-etcd v2.0.0+incompatible/go.mod h1:Jez6KQU2B/sWsbdaef3ED8NzMklzPG4d5KIOhIy30Tk=
+github.com/coreos/go-semver v0.2.0/go.mod h1:nnelYz7RCh+5ahJtPPxZlU+153eP4D4r3EedlOD2RNk=
+github.com/cpuguy83/go-md2man v1.0.10/go.mod h1:SmD6nW6nTyfqj6ABTjUi3V3JVMnlJmwcJI5acqYI6dE=
+github.com/d-sparks/ace-of-trades v0.0.0-20191022072657-7a5c9751d249 h1:GAwmlKnF/0+8fWWl36UkUL8agSRKzOOA10+gpkJZWfU=
```

Comment:
Two more ace-of-trades references here.

Reply:
looks like we use them in here
```
trading/simulate/simulate.go:   "github.com/d-sparks/ace-of-trades/trading"
trading/strategies/buy_and_hold/buy_and_hold.go:        "github.com/d-sparks/ace-of-trades/trading"
trading/strategies/buy_the_dip/buy_the_dip.go:  "github.com/d-sparks/ace-of-trades/trading"
trading/strategies/buy_the_dip/buy_the_dip.go:  "github.com/d-sparks/ace-of-trades/trading/strategies/buy_and_hold"
```
If i switch it to `gravy` I hit some errors. Are these packages expected to be functional?

Reply:
Gotcha. We can delete trading/strategies and trading/simulate whenever.